cmd/create/ocmrole/cmd.go

```diff
@@ -317,6 +320,7 @@ func run(cmd *cobra.Command, argv []string) {
 			env,
 			isAdmin,
 			managedPolicies,
+			autoLink,
```

Comment:
```suggestion
			confirm.Yes()
```
Suggestion if you move this to accept a bool you can just pass in `confirm.Yes()`